### PR TITLE
Change client-egress-l7-tls tests to sequential because flaky

### PR DESCRIPTION
**Pull request by youngnick**
Testing for flakes, not from a fork this time.

### CODE REVIEW

File: cilium-cli/connectivity/builder/client_egress_l7_tls_deny_without_headers.go [hunk 1]
```diff
@@ -14,7 +14,7 @@ type clientEgressL7TlsDenyWithoutHeaders struct{}
 func (t clientEgressL7TlsDenyWithoutHeaders) build(ct *check.ConnectivityTest, templates map[string]string) {
 	// Test L7 HTTPS interception using an egress policy on the clients.
 	// Fail to load site due to missing headers.
-	newTest("client-egress-l7-tls-deny-without-headers", ct).
+	newTest("seq-client-egress-l7-tls-deny-without-headers", ct).
```
Comment (viktor-kurchenko) [inline cilium-cli/connectivity/builder/client_egress_l7_tls_deny_without_headers.go:17]: Not sure if I understood properly, but if you want to run the test sequentially you need to move it from:
https://github.com/cilium/cilium/blob/main/cilium-cli/connectivity/builder/builder.go#L256

to:
https://github.com/cilium/cilium/blob/main/cilium-cli/connectivity/builder/builder.go#L289

Comment (youngnick) [reply cilium-cli/connectivity/builder/client_egress_l7_tls_deny_without_headers.go:17]: 🤦 Thanks!

File: cilium-cli/connectivity/builder/client_egress_l7_tls_headers.go [hunk 2]
```diff
@@ -19,7 +19,7 @@ func (t clientEgressL7TlsHeaders) build(ct *check.ConnectivityTest, templates ma
 }
 
 func clientEgressL7TlsHeadersTest(ct *check.ConnectivityTest, templates map[string]string, portRanges bool) {
-	testName := "client-egress-l7-tls-headers"
+	testName := "seq-client-egress-l7-tls-headers"
```
Comment (viktor-kurchenko) [inline cilium-cli/connectivity/builder/client_egress_l7_tls_headers.go:22]: @youngnick how about this test?

Comment (youngnick) [reply cilium-cli/connectivity/builder/client_egress_l7_tls_headers.go:22]: Damn, trying to multitask too hard this week.

**Review by viktor-kurchenko — APPROVED** [review]

**Pull request merged**